src/views/badge.j2

```diff
@@ -1,6 +1,7 @@
 <p><b>Badge #:</b> {{ Serial }}</p>
 <p><b>Name:</b> {{ name }}</p>
 <p><b>Rank:</b> {{ RankRole }}</p>
+<p><b>Unit Description:</b> {{ UnitDesc }}</p>
```

Comment:
I wanted to just comment on the entire file but I can't figure out how so I'm just commenting on this line.

I wonder if we might want to update the file name for the badge.j2 file to officer or person or something .j2 since this is more an object for a specific person and we have multiple ways to get back to this data object now, not just the badge lookup.

Reply:
Great point!